index.bs

```diff
@@ -488,13 +490,238 @@ partial interface NeuralNetworkContext {
     </div>
 </div>
 
+### gru ### {#api-neuralnetworkcontext-gru}
+Gated Recurrent Unit [[GRU]] recurrent network using an update gate and a reset gate to compute the hidden state that rolls into the output across the temporal sequence of the Network
+<script type=idl>
+enum RecurrentNetworkDirection {
+  "forward",
+  "backward",
+  "both"
+};
+
+enum RecurrentNetworkWeightLayout {
+  "zrn",  // update-reset-new gate ordering
+  "rzn"   // reset-update-new gate ordering
+};
+
+enum RecurrentNetworkActivation {
+  "relu",
+  "sigmoid",
+  "tanh"
+};
+
+partial interface NeuralNetworkContext {
+  sequence<Operand> gru(Operand input, Operand weight, 
+                        Operand recurrentWeight, Operand initialHiddenState,
```

Comment:
Should we make the initialHiddenState optional (with a value 0, if not specified)?

Reply:
good point. i'll look into it.

Reply:
Please take a look. `initialHiddenState` is now optional.